package.json

```diff
@@ -56,7 +53,7 @@
     "@babel/preset-react": "^7.0.0",
     "babel-core": "7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
-    "babel-jest": "^23.6.0",
+    "babel-jest": "^24.1.0",
```

Comment:
the 24 version of jest is a bit weird could you try to add some random snapshot test and see if its still working please?

Reply:
I have tested with this version and do not seem to have any issues.

Reply:
will do

Reply:
Changing to 23.6 until the 24 becomes more stable.